torch/distributed/rpc/backend_registry.py

```diff
@@ -356,14 +412,16 @@ def _tensorpipe_init_backend_handler(store, name, rank, world_size, rpc_backend_
                     {},
                     [],
                 )
+                api._init_rpc_states(agent)
 
                 try:
                     # TODO: Notify all workers in group this rank has joined and set devices and reverse_device_map
                     # This is a synchronous operation that completes once all existing ranks are updated
                     # _tensorpipe_check_remote_device_maps(agent, rpc_backend_options)
+                    _set_devices_and_reverse_device_map(agent, rank, rpc_backend_options.device_maps)
                     pass
                 except Exception:
-                    api.shutdown()
+                    # api.shutdown()
```

Comment:
what exceptions are we expecting here?

Reply:
I will remove this exception since we dont have shutdown implementation at the moment.